src/interfaces/IAminalStructs.sol

```diff
@@ -13,39 +13,26 @@ interface IAminalStructs {
     /**
      * @notice Visual trait configuration for an Aminal
      * @dev Contains the complete genetic visual profile of an Aminal
-     * @param backId Gene ID for background/back visual trait
-     * @param armId Gene ID for arm/limb visual trait
-     * @param tailId Gene ID for tail visual trait
-     * @param earsId Gene ID for ear visual trait
-     * @param bodyId Gene ID for body/torso visual trait
-     * @param faceId Gene ID for face visual trait
-     * @param mouthId Gene ID for mouth/expression visual trait
-     * @param miscId Gene ID for miscellaneous/accessory visual trait
+     * @dev Supports 1-10 genes with flexible ordering determined by placement metadata
+     * @dev A value of 0 for a geneId indicates an unused slot
+     * @param genes Array of up to 10 gene NFT IDs (0 = unused slot)
      */
     struct Visuals {
```

Comment:
Is it too annoying to rename this?

Reply:
Imagine it might break a lot of tests lol, but they are probably already broken.